example/kops_clusters.tf

```diff
@@ -21,10 +21,13 @@ module "cluster1" {
   internet_gateway_id       = "${aws_internet_gateway.public.id}"
   public_subnet_cidr_blocks = ["${local.cluster1_public_subnet_cidr_blocks}"]
   kops_dns_mode             = "private"
+  auditlog_logpath          = "/var/log/kube-apiserver-audit.log"
+  oidc_clientid             = "kubernetes"
+  oidc_issuerurl            = "https://accounts.google.com"
```

Comment:
The 2 examples in here should be as simple as possible to show how to use the module. Therefore I think we shouldn't be include OIDC stuff here by default. Could you just document how it works?